src/bitmessageqt/__init__.py

```diff
@@ -1295,24 +1373,23 @@ def _choose_ext(basename):
                 # elapsed time since the last sound was played
                 dt = datetime.now() - self.lastSoundTime
                 # suppress sounds which are more frequent than the threshold
-                if dt.total_seconds() < self.maxSoundFrequencySec:
-                    return
-
-            # the sound is for an address which exists in the address book
-            if category is sound.SOUND_KNOWN:
-                soundFilename = state.appdata + 'sounds/known'
-            # the sound is for an unknown address
-            elif category is sound.SOUND_UNKNOWN:
-                soundFilename = state.appdata + 'sounds/unknown'
-            # initial connection sound
-            elif category is sound.SOUND_CONNECTED:
-                soundFilename = state.appdata + 'sounds/connected'
-            # disconnected sound
-            elif category is sound.SOUND_DISCONNECTED:
-                soundFilename = state.appdata + 'sounds/disconnected'
-            # sound when the connection status becomes green
-            elif category is sound.SOUND_CONNECTION_GREEN:
-                soundFilename = state.appdata + 'sounds/green'
+                if not dt.total_seconds() < self.maxSoundFrequencySec:
```

Comment:
It seems to be a step backwards

Reply:
[pylint](https://stackoverflow.com/questions/43121340/why-is-the-use-of-lensequence-in-condition-values-considered-incorrect-by-pyli)

Reply:
Don't understand how it's related. You are nesting conditions adding extra indentation and then complain about 80 characters line width. The immediate return was added intentionally.